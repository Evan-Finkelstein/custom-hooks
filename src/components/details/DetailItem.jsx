import React from 'react';
import PropTypes from 'prop-types';

const DetailItem = ({ name, image, temperament, bred_for, }) => (
    <figure >
        <img style={{
            height: '50%',
            width: '50%',
        }} src={image} />
        <figcaption>{name}</figcaption>
        <figcaption>Temperament: {temperament}</figcaption>
        {/* <figcaption>Origin:{origin}</figcaption> */}
        <figcaption>Bred For:{bred_for}</figcaption>

    </figure>
);

DetailItem.propTypes = {
    name: PropTypes.string,
    url: PropTypes.string,
};

export default DetailItem;
