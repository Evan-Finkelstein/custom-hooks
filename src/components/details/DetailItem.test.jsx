import React from 'react';
import { render, cleanup } from '@testing-library/react';
import DetailItem from './DetailItem';

describe('DetailItem component', () => {
  afterEach(() => cleanup());
  it('renders DetailItem', () => {
    const { asFragment } = render(<DetailItem />);
    expect(asFragment()).toMatchSnapshot();
  });
});
