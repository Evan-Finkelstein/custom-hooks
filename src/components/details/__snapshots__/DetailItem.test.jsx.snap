// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailItem component renders DetailItem 1`] = `
<DocumentFragment>
  <figure>
    <img
      style="height: 50%; width: 50%;"
    />
    <figcaption />
    <figcaption>
      Temperament: 
    </figcaption>
    <figcaption>
      Bred For:
    </figcaption>
  </figure>
</DocumentFragment>
`;
